clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -335,6 +392,35 @@ func dumpAdditionalLogsOnPrometheusSetupFailure(k8sClient kubernetes.Interface)
 	klog.Info(string(s))
 }
 
+func getGceWindowsNodeNameFromKubernetesService(k8sClient kubernetes.Interface) (*gceNode, error) {
+	var nodeList *corev1.NodeList
+	f := func() error {
+		var err error
+		nodeList, err = k8sClient.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: "kubernetes.io/os=windows"})
+		return err
+	}
+
+	if err := client.RetryWithExponentialBackOff(client.RetryFunction(f)); err != nil {
+		return nil, err
+	}
+
+	if len(nodeList.Items) == 0 {
+		return nil, fmt.Errorf("no windows nodes available in kubernetes service")
+	}
+	// As we could scrape only one windows node as of now, so pick the first
```

Comment:
why only one node now?

Reply:
1. The windows node name would be passed into `targets: ["{{$WINDOWS_NODE_NAME}}:5000"]` in the additional-scrape-config.yaml file, and I don't know how to dynamically create slice of targets in yaml file yet. Will change it if I figured out later. 
2. And load test is for 1-node cluster, so scrape one node is sufficient as of now.

Reply:
Here is an example of slice variable in yaml - https://github.com/kubernetes/perf-tests/blob/3b2fdc539fb14ae7bb64e71370868fb1eb421e0b/clusterloader2/pkg/prometheus/manifests/master-ip/master-endpoints.yaml#L14

Reply:
Thanks, will support multiple nodes in following PR.